LakeExport.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+@author: simontopp
+"""
+#%%
+import time
+import ee
+import os
+#import pandas as pd
+#import feather
+
+## Initialize Earth Engine
+ee.Initialize()
+
+#Source necessary functions. We do this instead of 'import' because of EE quirk.
+exec(open('GEE_pull_functions.py').read())
+
+#water = ee.Image("JRC/GSW1_1/GlobalSurfaceWater").select('occurrence').gt(80)
+
+## Bring in EE Assets
+# Deepest point for CONUS PGDL Lakes
+# DP Code available at https://zenodo.org/record/4136755#.X5d54pNKgUE
+dp = (ee.FeatureCollection('users/sntopp/USGS/PGDL_lakes_deepest_point')
+  .filterMetadata('distance', "greater_than", 60))
+
+#CONUS Boundary
+us = ee.FeatureCollection("USDOS/LSIB_SIMPLE/2017")\
+    .filterMetadata('country_na', 'equals', 'United States')
+
+## WRS Tiles in descending (daytime) mode for CONUS
+wrs = ee.FeatureCollection('users/sntopp/wrs2_asc_desc')\
+    .filterBounds(us)\
+    .filterMetadata('MODE', 'equals', 'D')
+    
+pr = wrs.aggregate_array('PR').getInfo()
```

Comment:
what is "PR"?

Reply:
ohhh...Path-Row? i.e., a scene ID?

Reply:
Not scene ID, but scene footprint.  We do all the scenes in a given footprint in each round of the for loop.  This significantly speeds up computation time on EE servers.  Added a comment line to the code to clarify.